src/pickers/DatePicker.js

```diff
@@ -122,10 +120,8 @@ class openIMISDatePicker extends Component {
             format={secondCalendarFormatting}
             disabled={readOnly}
             value={this.state.value ? this.moveByOneDay(new Date(this.state.value)) : null}
-            {...((!!this.props.minDate || !!disablePast) && {
-              minDate: this.getMinDate(this.props.minDate, disablePast),
-            })}
-            {...(!!this.props.maxDate && { maxDate: this.props.maxDate })}
+            {...((!!minDate || disablePast) && this.getMinDate())}
```

Comment:
I am not sure if after this change getMindDate() is the best name. I was confused for a moment, thinking that code is incomplete. This suggest returning date only and not the whole prop. This could be reworked, but does not have to be, I'll leave it to your judgement, otherwise LGTM.